### PR TITLE
Add pipeline limitations to delivery report

### PR DESCRIPTION
**Pull request by ivadym**
## Description

https://github.com/Clinical-Genomics/project-planning/issues/446

### Added
- Pipeline specific limitations to delivery report

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### Test:
- Report includes pipeline specific limitations:

  Screenshot 2023-10-20 at 10 17 33

  Screenshot 2023-10-20 at 10 16 40

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: tests/mocks/mip_analysis_mock.py [hunk 1]
```diff
@@ -15,7 +16,7 @@ def create_mip_metrics_deliverables():
     return MIPMetricsDeliverables(**metrics_deliverables)
 
 
-class MockMipAnalysis:
+class MockMipAnalysis(MipAnalysisAPI):
     """Mock an MIP analysis object"""
```
Comment (henrikstranneheim) [inline tests/mocks/mip_analysis_mock.py:20]:
```suggestion
    """Mock an MIP analysis object."""
```

File: tests/store/conftest.py [hunk 2]
```diff
@@ -304,6 +304,35 @@ def store_with_an_application_with_and_without_attributes(
...
+            StoreConstants.TAG_APPLICATION_WITHOUT_ATTRIBUTES.value
+        ),
+        pipeline=Pipeline.BALSAMIC,
+    )
+    helpers.ensure_application_limitations(
+        store=store_with_an_application_with_and_without_attributes,
+        application=store_with_an_application_with_and_without_attributes.get_application_by_tag(
+            StoreConstants.TAG_APPLICATION_WITHOUT_ATTRIBUTES.value
+        ),
+        pipeline=Pipeline.MIP_DNA,
```
Comment (henrikstranneheim) [inline tests/store/conftest.py:331]: If only pipeline differs - use a loop

**Review by henrikstranneheim — APPROVED** [review]
Lovely 👍

**Comment by ivadym** [comment]
Deployed:

Screenshot 2023-10-23 at 14 06 23

**Pull request merged**